src/App.js

```diff
@@ -26,40 +28,44 @@ const App = () => {
   };
 
   return (
-    <form onSubmit={checkUserAnswers}>
-      {data.map(question => {
-        return (
-          <fieldset key={question.id}>
-            <legend>{question.text}</legend>
-
-            {question.answers.map(answer => {
-              return (
-                <div key={answer.id}>
-                  <input
-                    type="radio"
-                    id={`${question.id}.${answer.id}`}
-                    name={question.id}
-                    value={answer.id}
-                    onClick={handleAnswerSelect}
-                  />
-                  <label htmlFor={`${question.id}.${answer.id}`}>
-                    {answer.text}
-                  </label>
-                </div>
-              );
-            })}
-          </fieldset>
-        );
-      })}
-      {Object.keys(userAnswers).length === data.length && (
-        <button>Submit</button>
-      )}
-      {shouldDisplayScore && (
-        <h2>
-          You have got {score} out of {data.length}
-        </h2>
-      )}
-    </form>
+    <Container>
+      <Row>
+        <Col> <h1 style={{ textAlign: "center" }} >CYF Quiz</h1>  </Col>
+      </Row>
+      <Form onSubmit={checkUserAnswers}>
+        {data.map(question => {
+          return (
+            <Jumbotron>
+              <Form.Group key={question.id}>
+                <legend>{question.text}</legend>
+                {question.answers.map(answer => {
+                  return (
+                    <div key={answer.id}>
```

Comment:
This `div` doesn't do anything for us so we can remove it

Reply:
Would I use <Row> </Row> as we are using the key there ?

Reply:
No, `Row` is also unnecessary here. `Row` is used so that we can use the 12 column grid system. Here, we are not positioning multiple elements horizontally—we only have a single `Form.Check` element horizontally.

Since we are removing `div` then the key would go on its child which is the `Form.Check` below